README.md

```diff
@@ -38,9 +39,9 @@ Select 1:
 Describe the rules of scope in JavaScript.
 
 Your Answer:
-```text
 ```
-
+Scope is how Javascript functions are nested within one another. Global variables declared outside of a function may be utilized anywhere within the application. From there, the scope flows downward, and variables within the scope of one function will not be recognized in a separate function.
+```
```

Comment:
You're close! It's more about which variables can be accessed where. Please reference https://msdn.microsoft.com/library/bzt2dkta(v=vs.94).aspx